core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/internal/unmarshall/JsonUnmarshallerRegistry.java

```diff
@@ -24,7 +24,7 @@
  * Registry of {@link JsonUnmarshaller} implementations by location and type.
  */
 @SdkInternalApi
-final class JsonUnmarshallerRegistry extends AbstractMarshallingRegistry {
+public final class JsonUnmarshallerRegistry extends AbstractMarshallingRegistry {
```

Comment:
This is `@SdkInternal` so we can't use it outside of the model

Reply:
I didn't know the rule was that strict (but I can see from the annotation javadoc that is the definition) and it seemed like it was more efficient to use existing code. Discussion in separate thread.

Reply:
Have restored this file, and copied relevant code locally instead.